Source/WebInspectorUI/UserInterface/Workers/HeapSnapshot/HeapSnapshot.js

```diff
@@ -102,7 +104,7 @@ HeapSnapshot = class HeapSnapshot
         snapshotDataString = null;
 
         let {version, type, nodes, nodeClassNames, edges, edgeTypes, edgeNames, roots, labels} = json;
-        console.assert(version === 1 || version === 2, "Expect JavaScriptCore Heap Snapshot version 1 or 2");
+        console.assert(version === 1 || version === 2 || version === 3, version);
```

Comment:
It seems like the this assertion message got worse?
How about this?
```
`Expect JavaScriptCore Heap Snapshot version 1 - 3 but got ${version}`
```

Reply:
typically the `console.assert(...)` in Web Inspector use only the tested values as the message as engineers will only see these in inspector^2, meaning there'd be a link to the source code location where they can figure out what was intended by looking at the condition (i.e. the reason for providing the tested values is to know how exactly the assertion failed)

Reply:
ok.